libsolidity/analysis/TypeChecker.h

```diff
@@ -159,6 +159,9 @@ class TypeChecker: private ASTConstVisitor
 	void endVisit(Literal const& _literal) override;
 	void endVisit(UsingForDirective const& _usingForDirective) override;
 
+	/// Do not handle nor recurse into ImportDirective.
```

Comment:
I think this is not needed.

Reply:
Please try to import a library and rename it.

Reply:
in both cases, I think you meant the comments, or the override at all?

Reply:
I meant the full change (both lines) is not needed.

Reply:
```
import {A as B} from "test1.sol";
```
with `A` being an empty library

Does not cause an error, both with and without the visit here.